testgrid/specs/storage-migration.yaml

```diff
@@ -0,0 +1,767 @@
+      exit 1
+    fi
+- name: Migrate from Longhorn + Minio to OpenEBS + Minio
+  flags: "yes"
+  installerSpec:
+    kubernetes:
+      version: 1.24.x
+    containerd:
+      version: 1.6.x
+    flannel:
+      version: 0.21.x
+    registry:
+      version: 2.8.x
+    kotsadm:
+      version: latest
+    ekco:
+      version: latest
+    longhorn:
+      version: 1.3.x
+    minio:
+      version: latest
+  upgradeSpec:
+    kubernetes:
+      version: 1.24.x
+    containerd:
+      version: 1.6.x
+    flannel:
+      version: 0.21.x
+    registry:
+      version: 2.8.x
+    kotsadm:
+      version: latest
+    ekco:
+      version: latest
+    openebs:
+      version: 3.4.x
+      isLocalPVEnabled: true
+      localPVStorageClassName: local
+    minio:
+      version: latest
+  postInstallScript: |
+    source /opt/kurl-testgrid/testhelpers.sh
+    create_deployment_with_mounted_volume "migration-test" "default" "/data"
+    create_random_file_and_upload_to_deployment "migration-test" "default" "./test.data" "/data/test.data"
+    test_push_image_to_registry
+  postUpgradeScript: |
+    source /opt/kurl-testgrid/testhelpers.sh
+    sleep 120
+    download_file_from_deployment_and_compare "migration-test" "default" "./test.data" "/data/test.data"
+    pvc_uses_provisioner "migration-test" "default" "rook"
+    test_pull_image_from_registry
+    if kubectl get namespace/longhorn-system ; then
+      echo Longhorn namespace found after the upgrade.
+      exit 1
+    fi
+- name: Migrate from Rook 1.0.4 to OpenEBS (S3 disabled)
+  flags: "yes"
+  installerSpec:
+    kubernetes:
+      version: 1.19.x
+    containerd:
+      version: 1.6.x
+    flannel:
+      version: 0.21.x
+    registry:
+      version: 2.8.x
+    kotsadm:
+      version: latest
+    ekco:
+      version: latest
+    rook:
+      version: 1.0.4
+  upgradeSpec:
+    kubernetes:
+      version: 1.19.x
+    containerd:
+      version: 1.6.x
+    flannel:
+      version: 0.21.x
+    registry:
+      version: 2.8.x
+    kotsadm:
+      version: latest
+      disableS3: true
+    ekco:
+      version: latest
+    openebs:
+      version: 3.4.x
+      isLocalPVEnabled: true
+      localPVStorageClassName: local
+  postInstallScript: |
+    source /opt/kurl-testgrid/testhelpers.sh
+    create_deployment_with_mounted_volume "migration-test" "default" "/data"
+    create_random_file_and_upload_to_deployment "migration-test" "default" "./test.data" "/data/test.data"
+    test_push_image_to_registry
+  postUpgradeScript: |
+    source /opt/kurl-testgrid/testhelpers.sh
+    sleep 120
+    download_file_from_deployment_and_compare "migration-test" "default" "./test.data" "/data/test.data"
+    pvc_uses_provisioner "migration-test" "default" "rook"
+    test_pull_image_from_registry
+    if kubectl get namespace/rook-ceph ; then
+      echo Rook namespace found after the upgrade.
+      exit 1
+    fi
+- name: Migrate from Rook 1.4.x to OpenEBS (S3 disabled)
```

Comment:
This tile is == the other one. Could we have a name that let us know the diff?

Reply:
Kinda confusing but this one has `S3 disabled` while the other one does not.